app/services/submission_service.rb

```diff
@@ -319,6 +334,14 @@ def deliver_approval_notification(submission_id)
         .deliver_now
     end
 
+    def deliver_admin_submission_resubmitted_notification(submission_id)
+      submission = Submission.find(submission_id)
+      artist = Gravity.client.artist(id: submission.artist_id)._get
+
+      AdminMailer.submission_resubmitted(submission: submission, artist: artist)
```

Comment:
Is there a need for `artist` here? I've noticed that `AdminMailer.submission_resubmitted` only has `submission` argument`